services/frontend/src/components/SearchBar.js

```diff
@@ -0,0 +1,18 @@
+const SearchBar = () => {
```

Comment:
This is a good example of a component that (as of right now) doesn't accept and props and doesn't declare any state. It just has a `return()` statement with normal(-ish) html elements